rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -33,15 +34,15 @@ object SystemProcesses {
   private implicit class ProduceOps(
       res: Id[
         Either[OutOfPhlogistonsError.type, Option[
-          (TaggedContinuation, Seq[ListParWithRandomAndPhlos])
+          (ContResult[Par, BindPattern, TaggedContinuation], Seq[Result[ListParWithRandomAndPhlos]])
```

Comment:
That's an ugly type signature... Please extract type aliases clarifying the purpose of this type (I mean the whole type of `res`) and its components.